sphinx/environment/__init__.py

```diff
@@ -54,7 +54,7 @@
     'image_loading': 'link',
     'embed_stylesheet': False,
     'cloak_email_addresses': True,
-    'cve_base_url': 'https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-',
+    'cve_base_url': 'https://www.cve.org/CVERecord?id=CVE-',
```

Comment:
Does this need to be a docutils variable rather than a sphinx one?
Docutils variables are only really their for compatibility, a sphinx user should never have to deal with them